### PR TITLE
update Nomad_Moogle.lua (level 95)

### PR DESCRIPTION
**Pull request by ghost**
Add of level 95 to Fatalerror66 script. Thank you to him

### CODE REVIEW

File: scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua [hunk 1]
```diff
@@ -1,57 +1,145 @@
...
+   elseif(player:getQuestStatus(JEUNO,BEYOND_THE_STARS) == QUEST_ACCEPTED) then
+      player:startEvent(0x273d,0,1,3,1);
+   elseif(player:getQuestStatus(JEUNO,DORMANT_POWERS_DISLODGED) == QUEST_ACCEPTED) then
+      player:startEvent(0x273d,0,1,4,1);
+   elseif(player:getQuestStatus(JEUNO,BEYOND_INFINITY) == QUEST_ACCEPTED) then
+      player:startEvent(0x273d,0,1,5,1); -- player:startEvent(0x273d,0,1,6,1);
+   elseif(player:hasKeyItem(LIMIT_BREAKER) == true and player:getMainLvl() >= 75) then
+      player:startEvent(0x273d,0,1,0,0);
+   else
+      player:startEvent(0x273d,0,2,0,0);
```
Comment (TeoTwawki) [inline scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua]: Your indents need to be 4 spaces per indent. Please. I'm begging. You have 3. 2 indents should = 8 spaces.

File: scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua [hunk 2]
```diff
@@ -1,57 +1,145 @@
...
-	if(csid == 0x273D and option ~= -1) then
-		player:addKeyItem(LIMIT_BREAKER);
-		player:messageSpecial(KEYITEM_OBTAINED,LIMIT_BREAKER);
-	end
-	
-end;
\ No newline at end of file
+ local meritCount = player:getMeritCount();
+    
+ if(csid == 0x273d and option == -1) then
```
Comment (TeoTwawki) [inline scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua]: Missing indent here

**Comment by TeoTwawki** [comment]
And your pull request has merge conflicts :(

**Comment by AlexandreCA** [comment]
I have corrected the script but I not find how to remove the conflict :(

**Comment by bendangelo** [comment]
You have to pull darkstar into your branch and perform a merge.

**Comment by AlexandreCA** [comment]
f anyone can close post themselves and offer volunteer to show my script because I can not do ! :(

**Comment by TeoTwawki** [comment]
...How do did you delete and recreate your github account? Only seen one other person do that..

**Comment by AlexandreCA** [comment]
It was a mistake . I thought deleting my account it was going to remove my post. But that did nothing. I never managed to remove the problem of conflict.

**Comment by TeoTwawki** [comment]
Drop your file into a fresh clean clone of Darkstar. **I strongly suggest you do not try to do your commits through the website** and instead actually use git (and / or tortoisegit : https://tortoisegit.org/support/faq/#prerequisites ) on your pc, I swear to god its easy. There's an online tutorial even: https://try.github.io/

**Pull request closed without merging**